README.md

```diff
@@ -4,119 +4,80 @@
 >
 > This project is a proof of concept.
 
-This repository contains operational workflows for the [hubDashboard App](https://github.com/apps/hubDashboard).
-The workflow will generate predtimechart data and build a static site for each
-repository that installed the app.
+## Overview
 
-## Broad Architecture
+This repository contains scripts and re-usable workflows that can be used to
+build a hub dashboard that can be maintained separate from a hub.
 
-The hubDashboard App allows hub administrators to opt-in to an auto-generated
-dashboard. By default, when anyone installs the app on their dashboard
-repository, the dashboard will be built once on the initial build and then
-every day on a schedule defined in this repository. The role of the app in this
-workflow is twofold:
+## Usage
 
-1. provide a list of repositories that have it installed.
-2. provide credentials for our workflows to write to branches of the dashboard
+There are two ways to use these workflows:
 
-The workflow looks like this for each repository 
-(note the build processes are in parallel):
+1. via re-usable workflows (if you can run workflows in your repository)
+2. app-based runs (if you are unable to run workflows in your repository)
 
-```mermaid
-sequenceDiagram
-     participant GitHub
-     participant App
-     GitHub-->>App: Request installation name
-     App->>GitHub: Installed: "dashboard-repo"
-
-     create participant dashboard-repo
-     GitHub-->>dashboard-repo: fetch "dashboard-repo"
-     Note over GitHub: Builds static site to site/
-     GitHub-->>dashboard-repo: Read hub repository name from config
-     dashboard-repo->>GitHub: Hub Repository: "hub-repo"
-     create participant hub as hub-repository
-     GitHub-->>hub: fetch "hub-repo"
-     Note over GitHub: Builds Predtimechart data to data/
-     GitHub-->>App: Generate Access Token
-     App-->>GitHub: Access Toekn
-     GitHub->>dashboard-repo: Push site/ to gh-pages
-     GitHub->>dashboard-repo: Push data/ to ptc/data
-```
+For both options, you will make a copy of [the hub dashboard
+template](https://github.com/new?template_name=hub-dashboard-template&template_owner=hubverse-org).
 
-### Tools
+If you want to register [the hubDashboard
+app](https://github.com/apps/hubdashboard/installations/new), you can install
+it on the specific repository and then make a pull request to this repository
+that modifies `known-repos.json` that includes your repository.
```

Comment:
```suggestion
If you plan to use [the hubDashboard
app](https://github.com/apps/hubdashboard/installations/new), install it on the dashboard repository you just created. Then make a pull request here (`hub-dashboard-control-room`), adding the new dashboard repository name to `hub-repos.json`.
```

Feel free to discard the suggested wording (this is a hard sentence to write!), but it looks like `known-repos.json` should be `known-hubs.json`

Reply:
ah good point, but even in the suggestion you used `hub-repos.json` 🤪 

```suggestion
If you plan to use [the hubDashboard
app](https://github.com/apps/hubdashboard/installations/new), install it on the dashboard repository you just created. Then make a pull request here (`hub-dashboard-control-room`), adding the new dashboard repository name to `known-hubs.json`.
```